src/client/components/header/index.scss

```diff
@@ -0,0 +1,166 @@
+$boxshadow: rgba(0, 0, 0, .2);
+$navboxshadow: rgba(214, 214, 214, .5);
+$teamname: #d93738;
+$searchname: #868686;
+$navbackgroundcolor: #fff;
+$uploadnamecolor: #868686;
+
+.rootbody {
+  width: 100%;
```

Comment:
- do not chnage the root element within the header component
- root is 100% by default, if it is not, then we have to fix the bug, instead of hiding it